cluster-scope/overlays/ocp-staging/clusterissuers/patch.yaml

```diff
@@ -0,0 +1,3 @@
+- op: replace
```

Comment:
We typically name patches after the thing they are patching (so that the relationship is obvious), so this would be `letsencrypt-production-dns01_patch.yaml`.